src/interfaces/filing-interfaces/correction-interfaces.ts

```diff
@@ -9,6 +9,7 @@ interface CorrectionIF {
   correctedFilingType: string
   correctedFilingDate: string
   comment: string
+  type: string
```

Comment:
This should be an enum, since it's one of only 2 allowable values (plus null if not set). Refer to what Lewis did in Filings UI for this.

Reply:
https://github.com/bcgov/business-filings-ui/pull/356/files